bundles/org.openhab.binding.shelly/doc/images/UseCaseSmartRoller.md

```diff
@@ -0,0 +1,225 @@
+# Smartify Roller Shutters with openHAB and Shelly
+
+![](https://shelly.cloud/wp-content/uploads/2020/03/know_base.png)
+
+**How openHAB andShelly 2.5 provide an enhance experience when controlling rollers**
+
+## Use Case
+
+![](images/oh_image1.png)   ![](images/oh_image2.png)   ![](https://shelly.cloud/wp-content/uploads/2020/06/shelly25.jpg)
+
+More and more people have rollers, which are equipped with a motor and button control improving your convince when operating the shutter.
+
+This use case description covers the following aspects
+- Operate the roller manual as usual - don't loose some of those features when bringing in the Shelly
+- Use openHAB to control the roller (up/down/stop, positioning)
+- Utilize device's position favorites
+- Utilize safety features, which you get for free with the Shelly
+
+The first point is important: SmartHome is nice for us techies, but standard users should have to use the "standard mode", but also the "new world".
+Therefore a struct requirement: make if "compatible for my wife" and avoid "that sucks, I kill you"". Shutters not opening in the morning don't create fun :-)
+
+One more aspect is some kind of protection to avoid physical damage when something gets wrong. The Shellies provide some interesting features to do.
+
+This use case description might also apply to Venetian Blinds, but I don't have those and can't judge on that.
+Feel free to adapt the concept and report ideas and experiences to the [community](https://community.openhab.org/t/shelly-binding/56862/1702?u=markus7017) 
+
+## Technical pre-requisites
+
+The implement this use case you need
+- Roller motor can be attached to the Shelly
+For this how-to a roller with an electrical motor is required and will be control using A/C and a momentary switch.
+The same solutions works with other switch types (DC/AC, 2 button switch etc.).
+In this case you need to look into details and user a different Shelly configuration or adapt technical installation.
+Refer to the Alterco Shelly documentation and make sure to use proper technical installation and wiring.
+Important: Electrical installation should be performed only be people knowing what they do - failures could harm you!!
```

Comment:
I don't like these schoolmaster statements, but that's only a personal thing ;)

Reply:
Me too, but it's kind of a disclaimer. I don't want to get feedback "I got an electrical shock by following your instructions"